src/main/resources/com/google/api/codegen/java/settings.snip

```diff
@@ -162,8 +162,11 @@
   public static ApiClientHeaderProvider.Builder defaultApiClientHeaderProviderBuilder() {
     return ApiClientHeaderProvider.newBuilder()
         .setGeneratorHeader(DEFAULT_GAPIC_NAME, getGapicVersion())
-        .setApiClientHeaderLineKey("x-goog-api-client")
-        .addApiClientHeaderLineData(GrpcExtraHeaderData.getXGoogApiClientData());
+        @if xsettingsClass.transportProtocol == "GRPC"
+          .setApiClientHeaderLineKey("x-goog-api-client")
```

Comment:
We want the header line key for httpjson also.

Reply:
Can this be hardcoded as HTTP/1.1, given how stable that config will be?

Reply:
I just meant that x-goog-api-client should be included for discogapic clients. I don't fully understand your question.

Reply:
Oh gotcha. Moved the x-goog-api-client line out of the @if block.